src/csharp/Microsoft.Spark.Extensions.Delta.UnitTest/DeltaTableTests.cs

```diff
@@ -0,0 +1,180 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Spark.E2ETest;
+using Microsoft.Spark.E2ETest.Utils;
+using Microsoft.Spark.Extensions.Delta.Tables;
+using Microsoft.Spark.Sql;
+using Xunit;
+
+namespace Microsoft.Spark.Extensions.Delta.UnitTest
+{
+    /// <summary>
+    /// These tests will fail if the Delta Jar is not installed in the 
+    /// $SPARK_HOME/jars/ directory.
+    /// </summary>
+    [Collection(Constants.DeltaTestContainerName)]
+    public class DeltaTableTests
+    {
+        private readonly SparkSession _spark;
+
+        public DeltaTableTests(SparkFixture fixture)
+        {
+            _spark = fixture.Spark;
+        }
+
+        /// <summary>
+        /// Run the end-to-end scenario from the Delta Quickstart tutorial.
+        /// </summary>
+        /// <see cref="https://docs.delta.io/latest/quick-start.html"/>
+        [Fact]
```

Comment:
```suggestion
        [SkipIfSparkVersionIsLessThan(Versions.V2_4_3)]
```
Add V2_4_3 to [Versions.cs](https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark/Versions.cs)